### ISSUE TITLE
Latest build (1.13.0) breaks react native project

### ISSUE THREAD

**Issue by andresousa** [DataDog/dd-sdk-android-gradle-plugin#242]
**Describe what happened**
Build error: `app/build/generated/datadog/buildId/devRelease/datadog.buildId: Resource and asset merger: Duplicate resources`

**Steps to reproduce the issue:**
app `build.gradle` file:
```
plugins {
    (...)
    id("com.datadoghq.dd-sdk-android-gradle-plugin") version "1.+"
}

datadog {
    checkProjectDependencies = "none"
}

apply from: "../../node_modules/@datadog/mobile-react-native/datadog-sourcemaps.gradle"
(...)
```

**Describe what you expected:**
Build breaks with error: `app/build/generated/datadog/buildId/devRelease/datadog.buildId: Resource and asset merger: Duplicate resources`
Downgrading `com.datadoghq.dd-sdk-android-gradle-plugin` to `1.12.0` fixes the issue.

**Comment by 0xnm** [comment]
Hi @andresousa! Thanks for reporting it. Can you please share with us version of the our React Native SDK you are using?

**Comment by PetroPavlenko** [comment]
Hi I'm having same issue with
```
 "react-native": "0.72.5",
 "@datadog/mobile-react-native": "^2.2.0"
 ```
 Hope this helps

**Comment by 0xnm** [comment]
This should be fixed now in [1.14.0](https://github.com/DataDog/dd-sdk-android-gradle-plugin/releases/tag/1.14.0) release, please try it out.
